hazelcast-documentation/src/HazelcastSimulator/ExecutingASimulatorTest.md

```diff
@@ -1,185 +1,310 @@
+INFO  09:05:07 ==============================================================
+INFO  09:05:07 Current number of machines: 0
+INFO  09:05:07 Desired number of machines: 4
+INFO  09:05:07 GroupName: simulator-agent-hasan
+INFO  09:05:07 Username: simulator
+INFO  09:05:07 Using init script:/disk1/hasan/hazelcast-simulator-0.5/conf/init.sh
+INFO  09:05:07 JDK spec: oracle 7
+INFO  09:05:07 Hazelcast version-spec: outofthebox
+INFO  09:05:11 Created compute
+INFO  09:05:11 Machine spec: hardwareId=m3.medium,locationId=us-east-1,imageId=us-east-1/ami-fb8e9292
+INFO  09:05:18 Created template
+INFO  09:05:18 Login name to the remote machines: simulator
+INFO  09:05:18 Security group: 'simulator' is found in region 'us-east-1'
+INFO  09:05:18 Creating machines... (can take a few minutes)
+INFO  09:06:18     54.211.146.186 LAUNCHED
+INFO  09:06:18     54.166.1.79 LAUNCHED
+INFO  09:06:18     54.147.196.63 LAUNCHED
+INFO  09:06:18     54.144.235.111 LAUNCHED
+INFO  09:06:30     54.211.146.186 JAVA INSTALLED
+INFO  09:06:32     54.166.1.79 JAVA INSTALLED
+INFO  09:06:32     54.144.235.111 JAVA INSTALLED
+INFO  09:06:34     54.147.196.63 JAVA INSTALLED
+INFO  09:06:40     54.166.1.79 SIMULATOR AGENT INSTALLED
+INFO  09:06:40 Killing Agent on: 54.166.1.79
+INFO  09:06:40 Starting Agent on: 54.166.1.79
+INFO  09:06:40     54.211.146.186 SIMULATOR AGENT INSTALLED
+INFO  09:06:40 Killing Agent on: 54.211.146.186
+INFO  09:06:40     54.166.1.79 SIMULATOR AGENT STARTED
+INFO  09:06:40 Starting Agent on: 54.211.146.186
+INFO  09:06:40     54.211.146.186 SIMULATOR AGENT STARTED
+INFO  09:06:42     54.144.235.111 SIMULATOR AGENT INSTALLED
+INFO  09:06:42 Killing Agent on: 54.144.235.111
+INFO  09:06:42 Starting Agent on: 54.144.235.111
+INFO  09:06:43     54.144.235.111 SIMULATOR AGENT STARTED
+INFO  09:06:47     54.147.196.63 SIMULATOR AGENT INSTALLED
+INFO  09:06:47 Killing Agent on: 54.147.196.63
+INFO  09:06:47 Starting Agent on: 54.147.196.63
+INFO  09:06:47     54.147.196.63 SIMULATOR AGENT STARTED
+INFO  09:06:47 Pausing for machine warmup... (10 sec)
+INFO  09:06:57 Duration: 00d 00h 01m 49s
+INFO  09:06:57 ==============================================================
+INFO  09:06:57 Successfully provisioned 4 aws-ec2 machines
+INFO  09:06:57 ==============================================================
+INFO  09:06:57 Shutting down Provisioner...
+INFO  09:06:57 Done!
+```
 
-bar.class=yourgroupid.ExampleTest
-bar.maxKeys=5000
+Also you can see `agents.txt` was created automatically by provisioner in the working directory. This `agents.txt` file includes ips of the EC2 instances. You can take a glance at `agents.txt` via this command:
 
-```
+	less agents.txt
 
-This is useful if you want to run multiple tests sequentially, or tests in parallel using the `coordinator --parallel` option. Please see the [Coordinator section](#coordinator) for more information.
+As you can see, first column of ips is a public ips, second one is a private ips.
```

Comment:
Explain to the end user what public/private ip address is. Also explain that these are the ip addresses of the agents.